pyplanet/apps/contrib/live_rankings/__init__.py

```diff
@@ -109,27 +109,57 @@ async def handle_scores(self, players):
 						self.current_rankings.append(new_ranking)
 
 			self.current_rankings.sort(key=lambda x: x['score'])
-		elif 'rounds' in current_script or 'team' in current_script or 'cup' in current_script or \
-			'trackmania/tm_rounds_online' in current_script or 'trackmania/tm_teams_online' in current_script or 'trackmania/tm_cup_online' in current_script:
+		elif 'rounds' in current_script or 'team' in current_script or \
+			'trackmania/tm_rounds_online' in current_script or 'trackmania/tm_teams_online' in current_script:
 			for player in players:
 				if 'map_points' in player:
 					if player['map_points'] != -1:
-						new_ranking = dict(login=player['player'].login, nickname=player['player'].nickname, score=player['map_points'], points_added=0)
+						new_ranking = dict(login=player['player'].login, nickname=player['player'].nickname, score=player['map_points'], score_matchpoints=player['match_points'], points_added=0)
 						self.current_rankings.append(new_ranking)
 				elif 'mappoints' in player:
 					if player['mappoints'] != -1:
-						new_ranking = dict(login=player['login'], nickname=player['name'], score=player['mappoints'], points_added=0)
+						new_ranking = dict(login=player['login'], nickname=player['name'], score=player['mappoints'], score_matchpoints=player['matchpoints'], points_added=0)
 						self.current_rankings.append(new_ranking)
```

Comment:
Are we also implementing the matchpoints into Rounds/Team mode, or why are we passing that information here?
Also, while we're here: can't we do something about this if 'map_points' or 'mappoints' nonsense, surely we can move that into a variable or something?

Reply:
The 'map_points', 'mappoints' were already there. its based on 

https://github.com/PyPlanet/PyPlanet/blob/fc945224e57af49016ba280c41899dfd814db27a/pyplanet/apps/core/trackmania/callbacks.py#L68